gdx/src/com/badlogic/gdx/scenes/scene2d/ui/TextArea.java

```diff
@@ -73,6 +72,7 @@ protected void initialize () {
 		firstLineShowing = 0;
 		moveOffset = -1;
 		linesShowing = 0;
+		focusTraversal = false;
```

Comment:
Any reason for disabling focus traversal by default in TextArea?

Reply:
I'm thinking TextArea could provide focus traversal with TAB while ENTER creates a new line. That's how it appears to be on HTML / web pages.
I wouldn't know what is the default behaviour on mobile devices (specially regarding the ENTER key).

Reply:
Ok seems like i did not remind that.
Need to improve this, just implemented stupid the focus traversal suggestion in Issue #5908 

The default behaving of a TextArea on mobile is ENTER = new Line, TAB does not really exist.
On Desktops ENTER = new Line, TAB = focus traversal.
Need to check if it's mobile or desktop.

Reply:
Removed `focusTraversal = false;` again, solved it in a better way.